builtin checker findings for mslib/plugins/io/csv.py and mslib/utils/time.py

@matrss seems the builtin checker was updated?

I rather don't want to rename the csv io plugin, the time utils module can maybe refactored renamed to times (just plural?)

@joernu76 What do you think? 

I assume we need to solve this in stable.

```
Successfully installed flake8-7.0.0 flake8-builtins-2.4.0 mccabe-0.7.0 pycodestyle-2.11.1 pyflakes-3.2.0
mslib/plugins/io/csv.py:0:1: A005 the module is shadowing a Python builtin module "csv"
mslib/utils/time.py:0:1: A005 the module is shadowing a Python builtin module "time"
2     A005 the module is shadowing a Python builtin module "csv"
2
```

oh, running that locally

```
mslib/plugins/io/__init__.py:0:1: A005 the module is shadowing a Python builtin module "io"
mslib/plugins/io/csv.py:0:1: A005 the module is shadowing a Python builtin module "csv"
mslib/utils/time.py:0:1: A005 the module is shadowing a Python builtin module "time"

```

--- a/setup.cfg
+++ b/setup.cfg
@@ -37,6 +37,6 @@ omit =
 norecursedirs = .git .idea .cache
 
 [flake8]
-ignore = E124,E125,E402,W504
+ignore = E124,E125,E402,W504,A005
 max-line-length = 120
 exclude = mslib/msui/qt5/*.py, mslib/mscolab/migrations/*.py

